### PR TITLE
Move langpack name to its own property, expose it to the API (bug 1122256)

### PR DESCRIPTION
**Pull request by diox**
Needed for https://bugzilla.mozilla.org/show_bug.cgi?id=1122256

### CODE REVIEW

File: mkt/langpacks/tests/test_models.py [hunk 1]
```diff
@@ -44,6 +44,10 @@ def test_get_minifest_contents(self):
              'package_path': langpack.download_url,
              'developer': {'name': 'Mozilla'}})
 
+    def test_name(self):
+        langpack = LangPack(fxos_version='2.5.1')
+        eq_(langpack.name, u'English (US) language pack for Firefox OS 2.5.1')
```
Comment (diox) [inline mkt/langpacks/tests/test_models.py]: There is a comment above that also applies here: we should test this with translations, but need the weekly l10n extraction to be done and translations to be submitted first.

Comment (robhudson) [inline mkt/langpacks/tests/test_models.py]: Is there a bug? Or maybe add an extra `TODO`?

**Comment by robhudson** [comment]
r+wc

File: mkt/langpacks/tests/test_models.py (hunk 1 above)
Comment (diox) [inline mkt/langpacks/tests/test_models.py]: I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1129409

**Pull request merged**